src/js/msp/MSPHelper.js

```diff
@@ -2074,8 +2073,7 @@ MspHelper.prototype.crunch = function(code) {
                 }
                 if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
                     buffer.push8(FC.FILTER_CONFIG.dyn_lpf_curve_expo)
```

Comment:
```suggestion
                    buffer
                    .push8(FC.FILTER_CONFIG.dyn_lpf_curve_expo)
                    .push8(FC.FILTER_CONFIG.dyn_notch_count)
                    .push16(FC.FILTER_CONFIG.dyn_notch_bandwidth_hz);
                    .push8(FC.FILTER_CONFIG.dyn_notch_count);

```

Reply:
this looks good @haslinghuis but this does not follow the coding style we have followed so far in msp helper.

Reply:
True, I prefer the old way...